jobs/build_health.rb

```diff
@@ -135,4 +136,4 @@ def get_jenkins_build_health(build)
   Builds::BUILD_LIST.each do |build|
     send_event(build['id'], get_build_health(build))
   end
-end
+end
```

Comment:
I like having new lines at the ends of files 😄 Sorry, don't have any linting set up.